### PR TITLE
openapi-comment-parser updates

### PR DESCRIPTION
**Pull request by MaxShapiro32**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "express": "^4.17.1",
     "pino-http": "^5.1.0",
     "pino-pretty": "^4.0.0",
-    "openapi-comment-parser": "^0.1.0",
+    "openapi-comment-parser": "^0.3.0",
```
Comment (bourdakos1) [inline package.json]: use the latest version

File: src/app.js [hunk 2]
```diff
@@ -27,11 +25,8 @@ app.use(express.urlencoded({ extended: false }));
 
 // Setup Swagger.
 // Don't use `/` for swagger, it will catch everything.
-const spec = parseComments({
-  definition: baseDefinition,
-  paths: [path.join(__dirname, '**/*.?(js|yaml|yml)')],
-});
-app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(spec));
+const specs = openapi();
```
Comment (bourdakos1) [inline src/app.js]: I think we should use this to be on the safe side:
```js
const spec = openapi({ cwd: __dirname });
```

**Review by bourdakos1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (MaxShapiro32) [reply package.json]: @bourdakos1 there is a new version?

Comment (bourdakos1) [reply package.json]: uh

Comment (bourdakos1) [reply package.json]: @MaxShapiro32 `^v0.3.3`

Comment (MaxShapiro32) [reply package.json]: Updated

File: src/app.js (hunk 2 above)
Comment (MaxShapiro32) [reply src/app.js]: Updated

File: src/app.js [hunk 3]
```diff
@@ -27,11 +25,8 @@ app.use(express.urlencoded({ extended: false }));
 
 // Setup Swagger.
 // Don't use `/` for swagger, it will catch everything.
-const spec = parseComments({
-  definition: baseDefinition,
-  paths: [path.join(__dirname, '**/*.?(js|yaml|yml)')],
-});
-app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(spec));
+const specs = openapi({ cwd: __dirname });
```
Comment (bourdakos1) [inline src/app.js]: Nitpick, but can we keep this “spec”

File: src/{{.ServiceNameTitle}}.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+openapi: "3.0.3"
+info:
+  title: "Bee Travels {{.ServiceNameTitle}} Service"
+  version: "1.0.0"
+  description: "replace this with a real description of your service"
```
Comment (bourdakos1) [inline src/{{.ServiceNameTitle}}.yml]: We shouldn’t need any quotes for yaml

**Review by bourdakos1 — CHANGES_REQUESTED** [review]

File: src/{{.ServiceNameTitle}}.yml (hunk 4 above)
Comment (MaxShapiro32) [reply src/{{.ServiceNameTitle}}.yml]: Fixed

File: src/app.js (hunk 3 above)
Comment (MaxShapiro32) [reply src/app.js]: Changed

**Review by bourdakos1 — APPROVED** [review]
👍

**Pull request merged**